Tests/SwiftProtobufTests/Test_JSON_Group.swift

```diff
@@ -1,45 +0,0 @@
-// Test/Sources/TestSuite/Test_JSON_Group.swift - Exercise JSON coding for groups
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See http://swift.org/LICENSE.txt for license information
-// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-// -----------------------------------------------------------------------------
-///
-/// Google has not specified a JSON coding for groups.  But proto3 types
-/// can use proto2 types and vice-versa, so it seems reasonable to support
-/// JSON coding for proto2 groups.
-///
-/// This implementation treats groups just like messages for the purposes
-/// of JSON encoding.
-///
-// -----------------------------------------------------------------------------
-
-import Foundation
-import XCTest
-
-class Test_JSON_Group: XCTestCase, PBTestHelpers {
-    typealias MessageTestType = ProtobufUnittest_TestAllTypes
-
-    func testOptionalGroup() {
-        assertJSONEncode("{\"optionalgroup\":{\"a\":3}}") {(o: inout MessageTestType) in
-            var g = MessageTestType.OptionalGroup()
-            g.a = 3
-            o.optionalGroup = g
-        }
```

Comment:
Yes, good point. I've added this file back and changed the tests to decode and verify failure. I'll add tests for encoding (verify groups are dropped) later, because I'd like to clean that up at the same time that I clean up extension encoding.